rosdep/base.yaml

```diff
@@ -3916,7 +3921,6 @@ swi-prolog-http:
   arch: [swi-prolog]
   debian: [swi-prolog-http]
   fedora: [pl]
-  gentoo: [dev-lang/swi-prolog]
```

Comment:
> swi-prolog-http does not exist in debian

That's not good. We should figure out what it's supposed to be and push that in a separate merge request. First we should figure out if any packages are even using this dep anymore, though. It's possible this dep is a very outdated one, in which case we shouldn't worry about it.